spec/controllers/state_file/questions/submission_confirmation_controller_spec.rb

```diff
@@ -15,4 +15,33 @@
       expect(response_html).to have_text "state tax return is now submitted"
     end
   end
+
+  describe "nj veteran content" do
+    context "when nj and primary veteran" do
+      let(:intake) { create :state_file_nj_intake, :married_filing_jointly, primary_veteran: "yes", spouse_veteran: "no" }
+      render_views
```

Comment:
[dust] extremely inconsequential but you can do just one `render_views` at the top of this describe block